### PR TITLE
Transpose of a Matrix in C

### LINKED ISSUE #5581: Matrix Transpose in C
## 🚀 Feature

(A clear and concise description of what the feature is.)

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yes

## Pitch

Transpose of a Matrix in C

### PR DESCRIPTION
**Pull request by abhijeet141**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Transpose of a matrix in C

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Fixes #5581

### CODE REVIEW

**Comment by abhijeet141** [comment]
Respected Mentors please review my PR

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Few changes: 

- Add Multi-Line Comments 
- Add a beginner-friendly description at the top
- Use proper code Format 
- Remove extra lines of code and spaces
- Add sample I/O at the end of the Code
- Add Time and Space Complexity at the end of the Code   
- Add new line at the EOF

**Comment by abhijeet141** [comment]
> Few changes:
> 
> * Add Multi-Line Comments
> * Add a beginner-friendly description at the top
> * Use proper code Format
> * Remove extra lines of code and spaces
> * Add sample I/O at the end of the Code
> * Add Time and Space Complexity at the end of the Code
> * Add new line at the EOF

I will make all changes but please check i have added sample input and output at the end of the code

**Comment by ankitaggarwal23** [comment]
> > Few changes:
> > 
> > * Add Multi-Line Comments
> > * Add a beginner-friendly description at the top
> > * Use proper code Format
> > * Remove extra lines of code and spaces
> > * Add sample I/O at the end of the Code
> > * Add Time and Space Complexity at the end of the Code
> > * Add new line at the EOF
> 
> I will make all changes but please check i have added sample input and output at the end of the code

Okay

**Comment by abhijeet141** [comment]
@ankitaggarwal23  I have made all changes please review it once

File: C/math/matrix_transpose.c [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+Enter the number of columns in matrix = 3
+Enter the values in matrix  -->
+1   2   3
+4   5   6
+7   8   9
+Transpose of a matrix is -->
+1       4       7
+2       5       8
+3       6       9
+*/
```
Comment (ankitaggarwal23) [inline C/math/matrix_transpose.c]: rewrite it,

```
/*

Sample Output:

Enter the number of rows in matrix = 3
Enter the number of columns in matrix = 3

Enter the values in matrix  -->
1   2   3
4   5   6
7   8   9

Transpose of a matrix is -->
1       4       7
2       5       8
3       6       9

Time Complexity : O(mn) 
Space Complexity : O(mn)  
where,
	m = number of rows 
	n = number of columns

*/
```

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by abhijeet141** [comment]
@ankitaggarwal23 made changes please check

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**